packages/server-core/src/media/storageprovider/s3.storage.ts

```diff
@@ -30,13 +43,14 @@ export class S3Provider implements StorageProviderInterface {
   /**
    * Instance of S3 service object. This object has one method for each API operation.
    */
-  provider: AWS.S3 = new AWS.S3({
-    accessKeyId: config.aws.keys.accessKeyId,
-    secretAccessKey: config.aws.keys.secretAccessKey,
+  provider: S3Client = new S3Client({
+    credentials: {
+      accessKeyId: config.aws.keys.accessKeyId,
+      secretAccessKey: config.aws.keys.secretAccessKey
+    },
     endpoint: config.aws.s3.endpoint,
     region: config.aws.s3.region,
-    s3ForcePathStyle: true,
```

Comment:
Unless this has been tested and is not necessary, I think it needs to be kept it.

Reply:
s3ForcePathStyle has been removed from v3 client-s3 and maxRetries is renamed to maxAttempts. Added "forcePathStyle".